static/app/views/replays/detail/ai/replaySummaryContext.tsx

```diff
@@ -43,6 +44,7 @@ export function ReplaySummaryContextProvider({
         setupAcknowledgement.orgHasAcknowledged
     ),
   });
+  useEmitTimestampChanges();
```

Comment:
Removed the context that this was placed in, it wasn't being consumed. This is the most appropriate place to call this hook now.